src/uct/base/uct_md.c

```diff
@@ -268,9 +268,6 @@ ucs_status_t uct_iface_open(uct_md_h md, uct_worker_h worker,
 
     if (params->open_mode & UCT_IFACE_OPEN_MODE_DEVICE) {
         tl = uct_find_tl(md->component, params->mode.device.tl_name);
-    } else if ((params->open_mode & UCT_IFACE_OPEN_MODE_SOCKADDR_CLIENT) ||
-               (params->open_mode & UCT_IFACE_OPEN_MODE_SOCKADDR_SERVER)) {
-        tl = uct_find_tl(md->component, NULL);
```

Comment:
this is another change - you are deprecating UCT_IFACE_OPEN_MODE_SOCKADDR_CLIENT and UCT_IFACE_OPEN_MODE_SOCKADDR_SERVER flags in UCT API
better move it to another PR

Reply:
I've reverted this change. Waiting for CI result.